frontend/src/model/beta/ModelImages.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Button, Stack } from '@mui/material'
+import { useGetModelImages } from 'actions/model'
+import { useMemo, useState } from 'react'
+
+import { ModelInterface } from '../../../types/v2/types'
+import EmptyBlob from '../../common/EmptyBlob'
+import Loading from '../../common/Loading'
+import MessageAlert from '../../MessageAlert'
+import UploadNewImageDialog from '../../model/beta/images/UploadNewImageDialog'
+import ModelImageDisplay from '../beta/images/ModelImageDisplay'
+
+type AccessRequestsProps = {
+  model: ModelInterface
+}
+
+export default function ModelImages({ model }: AccessRequestsProps) {
+  const { modelImages, isModelImagesLoading, isModelImagesError } = useGetModelImages(model.id)
+
+  const [openUploadImageDialog, setOpenUploadImageDialog] = useState(false)
+
+  const modelImageList = useMemo(
+    () =>
+      modelImages.length ? (
+        modelImages.map((modelImage) => (
+          <ModelImageDisplay modelImage={modelImage} key={`${modelImage.namespace}-${modelImage.model}`} />
```

Comment:
Given that this comment (https://github.com/gchq/Bailo/pull/823/files#r1376614866) states that `modelImage.namespace` is the mode id and I'm guessing `modelImage.model` is also the model id, is `key={`${modelImage.namespace}-${modelImage.model}`}` a unique key across all images for the same model?